src/clj/rems/css/style_utils.clj

```diff
@@ -0,0 +1,79 @@
+(ns rems.css.style-utils
+  "CSS stylesheets are generated by garden automatically when
+  accessing the application on a browser. The garden styles can also
+  be manually compiled by calling the function `rems.css.styles/screen-css`.
+
+  For development purposes with Figwheel the styles are rendered to
+  `target/resources/public/css/:language/screen.css` whenever this ns is evaluated
+  so that Figwheel can autoreload them."
+  (:require [clojure.string :as str]
+            [clojure.test :refer [deftest is testing]]
+            [clojure.tools.logging :as log]
+            [compojure.core :refer [GET defroutes]]
+            [garden.color :as c]
+            [garden.core :as g]
+            [garden.def :refer [defkeyframes]]
+            [garden.selectors :as s]
+            [garden.stylesheet :as stylesheet]
+            [garden.units :as u]
+            [medley.core :refer [map-vals remove-vals]]
+            [mount.core :as mount]
+            [rems.config :refer [env]]
+            [rems.context :as context]
+            [ring.util.response :as response]))
```

Comment:
It looks like almost none of these requires are used. In case you want some tooling for this, the clj-kondo linter can point out unused required. An other option is the slamhound leiningen plugin, which can automatically remove unused requires from namespaces.

Reply:
The slamhound leiningen plugin is an excellent suggestion!